api/envoy/config/route/v3/route_components.proto

```diff
@@ -529,6 +529,13 @@ message RouteMatch {
   // against all the specified query parameters. If the number of specified
   // query parameters is nonzero, they all must match the *path* header's
   // query string for a match to occur.
+  //
+  // .. note::
+  //
+  //    If query parameters are used to pass request message fields when
+  //    grpc_json_transcoder is used, transcoded the message fields may be different
+  //    with the query parameters since the query parameters MUST be url_encoded,
```

Comment:
Instead of saying "different", just say that the fields have been '+' escaped for clarity.

Reply:
Done,  I added an example.